src/foam/nanos/ticket/SudoTicketRule.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.ticket',
+  name: 'SudoTicketOpenRule',
+  exends: 'foam.nanos.ruler.Rule',
+
+  implements: [
+    'foam.nanos.ruler.RuleAction'
+  ],
+
+  properties: [
+    {
+      name: 'description',
+      transient: true,
+      visibility: 'RO',
+      value: ''
+    },
+    {
+      class: 'String',
+      name: 'ruleGroup',
+      value: 'sudo',
+      visibility: 'RO',
+      // readPermissionRequired: true,
+      // writePermissionRequired: true
+    },
+    {
+      class: 'Enum',
+      of: 'foam.nanos.ruler.Operations',
+      name: 'operation',
+      value: 'CREATE',
+      visibility: 'RO',
+    },
+    {
+      class: 'Boolean',
+      nane: 'autoApprove',
+      value: false
+    },
+    {
+      name: 'assignToGroup',
+      class: 'Reference',
+      of: 'foam.nanos.auth.Group',
+      value: 'admin',
+      view: function(_, x) {
+        return foam.u2.view.ChoiceView.create({
```

Comment:
I think the default view for reference properties is more appropriate. It should work out of the box if you set `targetDAOKey: 'groupDAO'` on this property.

Reply:
Accidentally committed. Deleted.